core/src/main/resources/hudson/model/View/sidepanel.jelly

```diff
@@ -43,6 +47,47 @@ THE SOFTWARE.
         <l:task href="${rootURL}/${it.viewUrl}newJob" icon="icon-new-package icon-md" it="${app}" permission="${it.itemCreatePermission}" title="${%NewJob(it.newPronoun)}"/>
       </st:include>
 
+      <l:task href="#modal" icon="icon-new-package icon-md" title="Remodal Popup"/>
+      <!-- <st:include page="popup.jelly" optional="true" /> -->
```

Comment:
maybe you should make your popup.jelly as regular jelly file, but not only body part (add <jelly... to the begining)